File: hospitalApi/Models/Prescription.cs
using System;
using System.Collections.Generic;

namespace hospitalApi.Models;

public partial class Prescription
{
    public int Id { get; set; }

    public int FkMedicationId { get; set; }

    public int FkTreatmentId { get; set; }

    public int FkPrescribedByStaffId { get; set; }

    public double Doses { get; set; }

    public virtual Medication FkMedication { get; set; } = null!;

    public virtual Staff FkPrescribedByStaff { get; set; } = null!;

    public virtual Treatment FkTreatment { get; set; } = null!;
}

File: client/src/mocks/fixtures/prescription.ts
import type { HospitalApiDtosOutputsPrescriptionOutputDto } from "../../api";

export const mockPrescription: HospitalApiDtosOutputsPrescriptionOutputDto = {
    id: 8,
    fkMedicationId: 11,
    fkTreatmentId: 6,
    fkPrescribedByStaffId: 7,
    doses: 2,
};

export const mockPrescriptions: HospitalApiDtosOutputsPrescriptionOutputDto[] = [
    mockPrescription,
    { id: 9, fkMedicationId: 12, fkTreatmentId: 6, fkPrescribedByStaffId: 3, doses: 1 },
];
